docs/decisions/010-deprecations.md

```diff
@@ -46,7 +46,7 @@ Migration to new versions will be easier with clear upgrade paths.
 Testing environments may need to handle warning suppression.
 Additional maintenance effort is required to keep deprecation documentation up to date.
 
-### Breaking Changes without Deprecation
+### Breaking Changes Without Deprecation
```

Comment:
Is it okay that `with` is lowercase and `without` isn't?

Reply:
🤷